tests/test_smartstack_tools.py

```diff
@@ -255,8 +255,8 @@ def test_get_replication_for_instance(
     system_paasta_config,
 ):
     mock_mesos_slaves = [
-        {'hostname': 'host1', 'attributes': {'region': 'fake_region1'}},
-        {'hostname': 'host2', 'attributes': {'region': 'fake_region1'}},
+        {'hostname': 'host1', 'attributes': {'region': 'fake_region1', 'pool': 'default'}},
+        {'hostname': 'host2', 'attributes': {'region': 'fake_region1', 'pool': 'default'}},
```

Comment:
Can you expand the test a bit to test the new logic?

I think we could assert that `get_replication_for_all_services` is called with the correct host?

Reply:
But ship anyway if it is a real pain to do so for some reason :-)

Reply:
Yup good call, went too fast.